ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py

```diff
@@ -1000,25 +1000,24 @@ def test_paddle_lcm(
 # cumprod
 @handle_frontend_test(
     fn_tree="paddle.tensor.math.cumprod",
-    dtype_and_x=helpers.dtype_values_axis(
-        available_dtypes=helpers.get_dtypes("valid"),
-        safety_factor_scale="log",
-        large_abs_safety_factor=6,
-        small_abs_safety_factor=5,
-        min_num_dims=1,
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Ideally we want to test for all `"valid"` dtypes in our testing functions so as to not decrease the testing coverage. Do you think you can modify either the `unsupported_dtypes` or the `supported_dtypes` decorator to make this work